tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -903,3 +905,391 @@ def test_incorrect_folder_structure_raises(self):
             AssertionError, match="The main folder should contain at least one subfolder named 'Miniscope'."
         ):
             self.data_interface_cls(folder_path=folder_path)
+
+
+skip_on_darwin_arm64 = pytest.mark.skipif(
+    platform.system() == "Darwin" and platform.machine() == "arm64",
+    reason="The isx package is currently not natively supported on macOS with Apple Silicon. "
+    "Installation instructions can be found at: "
+    "https://github.com/inscopix/pyisx?tab=readme-ov-file#install",
+)
+skip_on_python_313 = pytest.mark.skipif(
+    sys.version_info >= (3, 13),
+    reason="Tests are skipped on Python 3.13 because of incompatibility with the 'isx' module "
+    "Requires: Python <3.13, >=3.9)"
+    "See:https://github.com/inscopix/pyisx/issues",
+)
+
+
+@skip_on_python_313
+@skip_on_darwin_arm64
+class TestInscopixImagingInterfaceMovie128x128x100Part1(ImagingExtractorInterfaceTestMixin):
+    """Test InscopixImagingInterface with movie_128x128x100_part1.isxd."""
+
+    data_interface_cls = InscopixImagingInterface
+    save_directory = OUTPUT_PATH
+    interface_kwargs = dict(
+        file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "inscopix" / "movie_128x128x100_part1.isxd")
+    )
+    optical_series_name = "OnePhotonSeries"
+
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_metadata(cls, request):
+        cls = request.cls
+
+        # Device metadata
+        cls.device_name = "Microscope"
+        cls.device_metadata = dict(name=cls.device_name, description="Inscopix Microscope")
+
+        # Imaging plane metadata
+        cls.imaging_plane_name = "ImagingPlane"
+        cls.imaging_plane_metadata = dict(
+            name=cls.imaging_plane_name,
+            description="Inscopix Imaging Plane",
+            device=cls.device_name,
+            optical_channel=[
+                dict(name="OpticalChannel", description="Inscopix Optical Channel", emission_lambda=np.nan)
+            ],
+        )
+
+        # One photon series metadata
+        cls.photon_series_metadata = dict(
+            name=cls.optical_series_name,
+            unit="n.a.",
+            dimension=[128, 128],
+            imaging_plane=cls.imaging_plane_name,
+        )
+
+        cls.ophys_metadata = dict(
+            Device=[cls.device_metadata],
+            ImagingPlane=[cls.imaging_plane_metadata],
+            OnePhotonSeries=[cls.photon_series_metadata],
+        )
+
+    def check_extracted_metadata(self, metadata: dict):
```

Comment:
Unless we define a method that in the interface that is called `get_metadata()` which we should it does not makes sense to override the parent methods here. This conversion is generic and the generic inherited methods should suffice.